packages/backend-ai/src/get_message.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  BedrockRuntimeClient,
+  ConverseCommand,
+  ConverseCommandInput,
+  Message,
+} from '@aws-sdk/client-bedrock-runtime';
+import {
+  ContentBlock,
+  GetMessageInput,
+  GetMessageOutput,
+  Tool,
+} from './types.js';
+
+/**
+ * Sends a message to the AI model and retrieves a response.
+ * This function leverages ConverseCommand (https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/bedrock-runtime/command/ConverseCommand/)
+ * @param params - The parameters for the message.
+ * @param params.systemPrompts - System prompts to guide the AI's behavior.
+ * @param params.messages - The conversation history.
+ * @param params.modelId - The ID of the AI model to use.
+ * @param [params.tools] - Optional tools that the AI can use.
+ * @returns The AI's response and associated metadata.
+ * @throws Throws an error if the AI's response does not contain a message.
+ */
+export const getMessage = async ({
+  systemPrompts,
```

Comment:
Question for @dbanksdesign maybe but would this be a nicer interface if it could just accept an array of strings - and we build out the `SystemContentBlock[]` in the function?